### ISSUE TITLE
[Bug] Prism.Forms for WPF platform fails executing.

### ISSUE THREAD

**Issue by urodolf** [PrismLibrary/Prism#1966]
### Description
We have created a solution for Xamrin.Foms and we have included Prism.Forms to use navigation and Prism events.

After running the application in Andriod we have compiled and executed the application in IOs and UWP without further problems, but when we have run the the application over WPF, an error is showed creating the Main application of the application that indicates that the ModuleManager does not implement ModuleDownloadProgressChanged of IModuleManager.

We have reviewed the Prism code and in IModuleManager we see a condition of the #if NET45 compiler for the ModuleDownloadProgressChanged that seems to be met for this project and therefore requires ModuleManager to implement it.

### Steps to Reproduce

1. Create a Xamarin.Forms solution.
2. Include Prism.Forms.
3. Create WPF platform project.
4. Execute WPF platform project.

### Basic Information

- Xamarin.Forms version: 4.4.0.991265
- Prism.Forms version: 7.2.0.1422
- IDE: Visual Studio 16.4.1
- Project WPF platform: .NET FW 4.7.2

**Comment by brianlagunas** [comment]
This is not a bug. WPF for XF is not support. I recommed creating a native WPF app instead.

**Comment by urodolf** [comment]
To test, we have downloaded the Prism code and included the ModuleDownloadProgressChanged event in Prism.Forms and removed the condition of the #if NET45 compiler from IModuleManager. We have generated a nuget and we have tried to include it in the solution and now everything works without problem.

Logically this is not a solution, since the ModuleDownloadProgressChanged event is not used in Prism.Forms, but I hope this information will help you.
